view/frontend/web/js/view/cart/product-protection-simple-offer.js

```diff
@@ -59,7 +59,7 @@ define(['cartUtils', 'extendSdk', 'ExtendMagento'], function (cartUtils, Extend,
     Extend.config({ storeId: config[0].extendStoreUuid, environment: config[0].activeEnvironment })
 
     Extend.buttons.renderSimpleOffer(
-      '#product_protection_offer_' + config[0].selectedProductSku,
+      '#product_protection_offer_' + encodeURI(config[0].selectedProductSku),
```

Comment:
I see you have to use a different function between phtml and js, have you confirmed the result is the same? I imagine url encoded is url encoded, just a little concerning.

Reply:
https://stackoverflow.com/questions/7013643/urlencode-from-php-in-javascript#:~:text=6%20Answers&text=There%20is%20no%20function%20quite,to%20rawurlencode()%20%3A%20encodeURIComponent()%20.&text=My%20code%20is%20the%20JS,based%20on%20PHP's%20source%20code).&text=JS%20function%20equivalent%20to%20PHP's%20urldecode.

swapping to rawurlencode